lib/debug/synonyms.rb

```diff
@@ -0,0 +1,69 @@
+module Debug
+  module Synonyms
+    class OldModel
+      attr_reader :client
+      attr_reader :index
+
+      def initialize(index: "govuk", client: Services.elasticsearch)
+        @client = client
+        @index = index
+      end
+
+      def search(query)
+        search_query = {
+          query: {
+            multi_match: {
+              "query" => query,
+              "fields" => %w(title^1000 description)
+            }
+          },
+          highlight: {
+            "fields" => { "title" => {}, "description" => {} },
+            "pre_tags" => ["\e[32m"],
```

Comment:
would prefer to pass this value in as a param or have it as a constant - same for the post_tags